### ISSUE TITLE
No data for names field in places data?

### ISSUE THREAD

**Issue by drwjf** [OvertureMaps/data#134]
After upgrade to `2024-03-12-alpha.0`,  it looks like all `names` field value changed to `common=null, official=null, alternate=null, short=null}`.

For example:

SELECT
       id,
       geometry, -- WKB
       CAST(names AS JSON),
       CAST(addresses AS JSON),
       CAST(categories AS JSON),
       CAST(bbox AS JSON),
       CAST(websites AS JSON),
       CAST(phones AS JSON)
FROM
       places
WHERE
        id = '08f2832184bb409c03ddccc7814b03a2'

result is:

"08f2832184bb409c03ddccc7814b03a2","00 00 00 00 01 c0 5e 35 87 bd cf 03 08 40 43 5d 79 d0 a6 76 21","{""common"":null,""official"":null,""alternate"":null,""short"":null}","[{""freeform"":""279 Placerville Dr"",""locality"":""Placerville"",""postcode"":""95667-3911"",""region"":""CA"",""country"":""US""}]","{""main"":""spas"",""alternate"":[""beauty_and_spa"",""beauty_salon""]}","{""minx"":-120.83641,""maxx"":-120.83641,""miny"":38.73028,""maxy"":38.73028}","[""http://lotusmassagespa.com""]","[""+15306260888""]"

**Comment by drwjf** [comment]
The result in `2023-11-14-alpha.0` looks good, it was

"141001552898634","00 00 00 00 01 c0 5e 35 87 bf e7 e1 fc 40 43 5d 79 df bf ee 98","{""common"":[{""value"":""Placerville Massage & Spa"",""language"":""local""}],""official"":null,""alternate"":null,""short"":null}","[{""freeform"":""279 Placerville Dr"",""locality"":""Placerville"",""postcode"":""95667-3911"",""region"":""CA"",""country"":""US""}]","{""main"":""spas"",""alternate"":[""beauty_and_spa"",""beauty_salon""]}","{""minx"":-120.8364105,""maxx"":-120.8364105,""miny"":38.7302818,""maxy"":38.7302818}"

(id is different, GSP address is same)

**Comment by jwass** [comment]
Hi @drwjf - we actually changed the names schema starting in the February release. Details here: https://docs.overturemaps.org/overview/feature-model/names

I queried the names in the latest version using duckdb:
```
D select names from read_parquet('s3://overturemaps-us-west-2/release/2024-03-12-alpha.0/theme=places/type=place/*') limit 10;
100% ▕████████████████████████████████████████████████████████████▏
┌────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│                                                                           names                                                                            │
│ struct("primary" varchar, common map(varchar, varchar), rules struct(variant varchar, "language" varchar, "value" varchar, "at" double[], side varchar)[]) │
├────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ {'primary': Mount Boyd, 'common': NULL, 'rules': NULL}                                                                                                     │
│ {'primary': Erickson Glacier, 'common': NULL, 'rules': NULL}                                                                                               │
│ {'primary': Mount Cromie, 'common': NULL, 'rules': NULL}                                                                                                   │
│ {'primary': O'Leary Peak (Antarctica), 'common': NULL, 'rules': NULL}                                                                                      │
│ {'primary': Dufek Coast, 'common': NULL, 'rules': NULL}                                                                                                    │
│ {'primary': Pallid Peak, 'common': NULL, 'rules': NULL}                                                                                                    │
│ {'primary': Wilson Portal, 'common': NULL, 'rules': NULL}                                                                                                  │
│ {'primary': Anderson Heights, 'common': NULL, 'rules': NULL}                                                                                               │
│ {'primary': Kosco Glacier, 'common': NULL, 'rules': NULL}                                                                                                  │
│ {'primary': Mincey Glacier, 'common': NULL, 'rules': NULL}                                                                                                 │
├────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│                                                                          10 rows                                                                           │
└────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
```

**Comment by drwjf** [comment]
Thanks!

SQL language in 
https://github.com/OvertureMaps/data/blob/b278afb12ca42c5e081b5c4253bfd543db56507d/athena_setup_queries.sql#L193

still uses the deprecated format.

If it is needed to update, I would like to raise a PR to change it.

**Comment by jwass** [comment]
Ugh - yeah that is out of date. 

If you're using Athena we have updated instructions here https://labs.overturemaps.org/how-to/accessing-data/cloud-services/#amazon-athena-aws that allow you to add the Overture release data catalog which will just stay up to date with all releases without much work for you. The DDL statements there are up-to-date.

**Comment by jwass** [comment]
@jamesdaus-msft @jenningsanderson 

@drwjf pointed out the athena-setup-queries is out-of-date. Should that file just be removed in favor of  https://labs.overturemaps.org/how-to/accessing-data/cloud-services/#amazon-athena-aws? Or update it to match? Not sure what's best here.

**Comment by jamesdaus-msft** [comment]
Totally agree, and this is addressed in #133 
I think it would be good if we update that to remove the other files and merge soon to avoid further confusion.

**Comment by jenningsanderson** [comment]
This issue appears resolved, closing. Feel free to re-open if necessary
